### ISSUE TITLE
Exit status when deleting orphaned routes fails is: success!

### ISSUE THREAD

**Issue by pronoiac** [cloudfoundry/cli#978]
- Command, What occurred, incl. error and/or stack trace

```
$ cf delete-orphaned-routes -f && echo "Boo, hiss"
Getting routes as admin ...

Deleting route e9d1ed51-9bca-4745-8c76-4bab510ac7dc.apps.redacted.com...
Deleting route b1ba4da0-ea2e-41a4-8441-c1c3bbc7dd37.apps.redacted.com...
Deleting route 79015260-9614-430b-81ed-ef53d7a6ce8b.apps.redacted.com...
Deleting route 096f3844-4387-47c1-bb08-fa91cc5dd60e.apps.redacted.com...
Deleting route 06b9db6a-7cc9-489e-94ce-57ba329d3131.apps.redacted.com...
FAILED
An unknown error occurred.
OK
Boo, hiss
```
- What you expected to occur
  I expected it to error out with an exit status reflecting an error, not success. 
- CLI Version  
  cf version 6.22.1+6b7af9c-2016-09-24
- Platform & Shell Details
  Seen on Mac OS X 10.11.6 with iTerm 2

**Comment by dkoper** [comment]
@pronoiac I'm also concerned about the "unknown error". If you can reproduce this, can you run it with trace enabled and attacj it (after redacting)?

**Comment by dkoper** [comment]
This issue has now been resolved in cf CLI 6.23.0.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM
